docs/backends/googlecloudotlp.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Google Cloud (OTLP)'
+description: 'Configuring the Google Cloud (OTLP) backend (Managed)'
+sidebarTitle: 'Google Cloud (OTLP)'
+icon: 'signal-stream'
+---
+
+### Getting Started
+
+<img src="https://d15jtxgb40qetw.cloudfront.net/gcp.svg" alt="googlecloudotlp" className="not-prose h-20" />
+
+{/*
+    Add custom content here (under this comment)...
+
+    e.g.
+
+    **Creating Account**<br />
+    Go to the **[🔗 website](https://odigos.io) > Account** and click **Sign Up**
+
+    **Obtaining Access Token**<br />
+    Go to **⚙️ > Access Tokens** and click **Create New**
+
+    !! Do not remove this comment, this acts as a key indicator in `docs/sync-dest-doc.py` !!
+    !! START CUSTOM EDIT !!
+*/}
+
+### Google Cloud Telemetry API
+
+This destination uses the [Google Cloud Telemetry (OTLP) API](https://cloud.google.com/stackdriver/docs/reference/telemetry/overview)
+to send data as OTLP to GCP Observability using an OTLP/HTTP exporter. It is different from the
+[Google Cloud Monitoring destination](https://docs.odigos.io/backends/googlecloudmonitoring)
+which uses the [googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/d9769f7c1a0f3bea82c711af551d9b5b44d69e6f/exporter/googlecloudexporter).
+
+Odigos currently supports Standard (non-Autopilot) GKE clusters.
```

Comment:
It actually supports any cluster now, maybe we should rephrase this part to make it more obvious that this only relates to the automatic authentication?

Reply:
on GCP we still don't support Autopilot, but I made it clearer that this is only referring to GCP clusters